pkg/rancher-desktop/components/UpdateStatus.vue

```diff
@@ -36,6 +36,27 @@
         <span v-else></span>
       </template>
     </card>
+    <card v-else-if="unsupportedUpgradeAvailable" ref="osUpdateRequired" :show-highlight-border="false">
+      <template #title>
+        <div class="type-title">
+          <h3>Operating System Upgrade Required</h3>
```

Comment:
Technically we're only reporting that an upgrade is not support, not that it's an OS upgrade (though the current state is). For example, if we can in the future detect that we need some new hardware feature in the next version…